apps/analysis/tests/test_apis.py

```diff
@@ -583,3 +584,99 @@ def test_analysis_overview_in_project(self):
         self.authenticate(user2)
         response = self.client.get(url)
         self.assert_403(response)
+
+    def test_post_discarded_entries_in_analysis_pillar(self):
+        user = self.create_user()
+        entry = self.create_entry()
+        project = self.create_project()
+        project.add_member(user)
+        analysis = self.create(Analysis, project=project)
+        pillar1 = self.create(AnalysisPillar, analysis=analysis)
+        data = {
+            'entry': entry.id,
+            'tag': DiscardedEntry.TagType.REDUNDANT
+        }
+        url = f'/api/v1/analysis-pillar/{pillar1.id}/discarded-entries/'
+        self.authenticate(user)
+        response = self.client.post(url, data)
+        self.assert_201(response)
+        self.assertEqual(response.data['analysis_pillar'], pillar1.id)
+        self.assertEqual(response.data['entry'], entry.id)
```

Comment:
Also, users shouldn't be able to discard an entry to which they don't have permission too.